bindkey/index.md

```diff
@@ -4,4 +4,4 @@ This type of credential lets the controller of an AID make a formal, public decl
 
 Normally, we expect key control to be expressed in the KEL. However, there exist use cases where such things must be externalized. For example, the email security standards SPF, DKIM, and DMARC deal with how a company can force all email from their domain to meet a higher standard of proof (to eliminate phishing and other spoofing). These standards want a company to use RSA keys. We don't really want AIDs to be based on RSA crypto, but it could be useful for orgs to announce to the world that they are using an RSA key for these email security standards.
 
-Schema is in [`schema.json`](schema.json); an informally example (that looks plausible but won't verify) is in [`example.json`](example.json).
\ No newline at end of file
+Schema is in [`bindkey.schema.json`](bindkey.schema.json); an informally example (that looks plausible but won't verify) is in [`example-bindkey.json`](example-bindkey.json).
```

Comment:
I'm realizing that I somehow used bad grammar on this line ("informally example"); as long as you're changing the line, will you change that word too?